### ISSUE TITLE
other languages in the interface

### ISSUE THREAD

**Issue by OmlineEditor** [msoultanidis/quillnote#154]
please make it possible to select other languages in the program interface. I can help with the translation into my own Russian language

**Comment by dankxylese** [comment]
Yes please add this. You can host the translation effort on crowdin or weblate. I can help with Ukrainian, and I'm others are willing with other languages

**Comment by dankxylese** [comment]
> please make it possible to select other languages in the program interface. I can help with the translation into my own Russian language

You have to translate the file in 
quillnote/app/src/main/res/values/strings.xml 

Make a fork.

Inside it make a new folder in 
quillnote/app/src/main/res/ 

And call it "values-ru" for Russian. 

Put the translated strings.xml in there and make a pull request

---

Language is still not selectable, which would be nice, but if you change your system language the app should follow.

**Comment by OmlineEditor** [comment]
by default, the language is automatically selected from the system, but you need to make it possible to manually select the interface language.
